packages/react/src/components/auth/LoginForm.js

```diff
@@ -87,7 +97,49 @@ export default function LoginForm() {
           <Box css={fieldCSS}>
             <Box css={fieldLabel}>Password</Box>
             <Box css={fieldRow}>
-              <Input type="password" onChange={handleEditPassword} />
+              <Input
+                type={showPassword ? 'text' : 'password'}
+                onChange={handleEditPassword}
+              />
+              <Box
+                type="button"
+                css={passwordEyeCss}
+                onClick={handleTogglePassword}
+              >
+                {showPassword ? (
+                  <svg
```

Comment:
Lets move the svgs to a different file

Reply:
hi, do we have any common assets folder or some other rules for adding svg's ?

Reply:
@sidmohanty11, can you please clarify so that I can move forward with this issue.
Thanks!